envoy/upstream/upstream.h

```diff
@@ -528,11 +528,22 @@ class PrioritySet {
 };
 
 /**
- * All cluster stats. @see stats_macros.h
+ * All cluster config update related stats.
```

Comment:
I think some comments here about the rationale for the partitioning would help.

In particular, what I think you mean to have here is a block of stats you expect to be incremented for clusters that see zero traffic.

Whereas ALL_CLUSTER_STATS (which should be re-named as that's now a lie :) are stats that you expect to only be incremented when a cluster sees traffic. Is that right?

Reply:
sounds good. I will more details in comment.